src/java.base/share/classes/java/lang/CharacterData.java

```diff
@@ -72,23 +72,15 @@ static final CharacterData of(int ch) {
         if (ch >>> 8 == 0) {     // fast-path
             return CharacterDataLatin1.instance;
         } else {
-            switch(ch >>> 16) {  //plane 00-16
-            case(0):
-                return CharacterData00.instance;
-            case(1):
-                return CharacterData01.instance;
-            case(2):
-                return CharacterData02.instance;
-            case(3):
-                return CharacterData03.instance;
-            case(14):
-                return CharacterData0E.instance;
-            case(15):   // Private Use
-            case(16):   // Private Use
-                return CharacterDataPrivateUse.instance;
-            default:
-                return CharacterDataUndefined.instance;
-            }
+            return switch (ch >>> 16) {  //plane 00-16
+                case 0 -> CharacterData00.instance;
+                case 1 -> CharacterData01.instance;
+                case 2 -> CharacterData02.instance;
+                case 3 -> CharacterData03.instance;
+                case 14 -> CharacterData0E.instance; // Private Use
```

Comment:
Plane 14 is not `private use`

Reply:
Hi Naoto. Well spotted. I've corrected that now. See a8706b0